apps/storybook/src/TiledHeatmap.stories.tsx

```diff
@@ -0,0 +1,208 @@
+import {
+  getLevelsSize,
+  PanMesh,
+  VisCanvas,
+  ZoomMesh,
+  TiledHeatmap,
+  TilesApi,
+} from '@h5web/lib';
+import type { ColorMap, Domain, Size } from '@h5web/lib';
+import { ScaleType } from '@h5web/shared';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { clamp } from 'lodash';
+import ndarray from 'ndarray';
+import { useMemo } from 'react';
+import { createFetchStore } from 'react-suspense-fetch';
+import type { Vector2 } from 'three';
+
+import FillHeight from './decorators/FillHeight';
+
+// See https://en.wikipedia.org/wiki/Mandelbrot_set
+function mandelbrot(
+  iterations: number,
+  xDomain: Domain,
+  yDomain: Domain,
+  size: Size
+) {
```

Comment:
When it's easy-enough to do, it's usually good to add an explicit return type to help TS (and readability).